kubernetes/active-passive/check-readiness.sh

```diff
@@ -15,11 +15,11 @@ podhealth() {
   curl -I $activepod
 
   # if curl fails then we need to be active otherwise stay passive
-  if [[ $? -ne 0 ]]
+  if [[ "$?" -eq "0" ]]
```

Comment:
why do you need to make strings out of these ?

Reply:
because b a s h